linera-core/src/remote_node.rs

```diff
@@ -246,54 +242,16 @@ impl<N: ValidatorNode> RemoteNode<N> {
         }
     }
 
-    /// Returns the list of certificate hashes on the given chain in the given range of heights.
-    /// Returns an error if the number of hashes does not match the size of the range.
```

Comment:
We could still check in this function that we got the certificates with the correct heights and chain ID.